src/analyze/gui/mainwindow.cpp

```diff
@@ -404,6 +406,26 @@ MainWindow::MainWindow(QWidget* parent)
     m_ui->menu_File->addAction(m_closeAction);
     m_quitAction = KStandardAction::quit(qApp, SLOT(quit()), this);
     m_ui->menu_File->addAction(m_quitAction);
+    QShortcut* shortcut = new QShortcut(QKeySequence(QKeySequence::Copy), m_ui->stacksTree);
+    connect(shortcut, &QShortcut::activated,
+            [&]()
+    {
+      QTreeView* view = m_ui->stacksTree;
+      if (view->selectionModel()->hasSelection())
```

Comment:
style: join next line

Reply:
I don't know what you mean.

Reply:
make it:

```
if () {
   ...
}
```

instead of

```
if ()
{
    ...
}
```

Reply:
ok. You could provide a clang-format config file if you want contributors to easily comply with your style.

Reply:
very true, I'll have to do that eventually